pkg/crc/preflight/preflight_checks_darwin.go

```diff
@@ -84,7 +84,7 @@ func fixVirtualBoxInstallation() (bool, error) {
 }
 
 func checkResolverFilePermissions() (bool, error) {
-	err := unix.Access(resolverFile, unix.R_OK | unix.W_OK)
+	err := unix.Access(resolverFile, unix.R_OK|unix.W_OK)
 	if err != nil {
```

Comment:
These changes are unrelated to this commit/this patch series.

Reply:
there are due to missed `gofmt` by a previous commit. this is something that needs to be fixed... just happened to be in my commit as I always run this befor e`git commit`.